src/parse_fasta.py

```diff
@@ -107,8 +107,3 @@ def map_reads(FASTA_1, FASTA_2):
             if index_list:
                 dict[sequences[0][s]].update({genomes[0][g]: index_list})
     return dict
-
-
-if __name__ == '__main__':
```

Comment:
We only need this in scripts that you want to directly run from the command-line, i.e., entry points into your code. While you may have multiple entry points in a package, in most simple cases you will have only one - a command-line interface with some application-level code that calls your functions from your modules wherever needed to provide one or more behaviors. For reasons of portability, modules that are just collections of functions or classes and methods should not be entry points and therefore shouldn't have this.

See here for what the `if __name__ == '__main__':` pattern actually does: https://stackoverflow.com/questions/419163/what-does-if-name-main-do